package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@okta/okta-react",
-  "version": "4.1.0",
+  "version": "4.1.1",
```

Comment:
version bump is not needed here. As it a new feature, let's release it as 4.1.0 (the version in master reflects the next minor version)